task_manager/tests/conftest.py

```diff
@@ -18,6 +20,16 @@ def service_user() -> ServiceUser:
     )
 
 
+@pytest.fixture
+def status() -> Status:
+    return Status.objects.create(name="active")
+
+
+@pytest.fixture
+def task(service_user: ServiceUser, status: Status) -> Task:
+    return Task.objects.create(name="deploy", status=status, author=service_user)
```

Comment:
I might be old-fashioned, but I prefer creating entities for tests explicitly - so I'd convert these fixtures to some sort of `helper` functions and call them directly in tests
this approach is not always possible - e.g. when session fixture creates user behind the scenes

feel free to keep it as it is :)